### ISSUE TITLE
"Element in sequential focus order has visible focus" (oj04fd)

### ISSUE THREAD

**Issue by susanatx** [act-rules/act-rules.github.io#1599]
I think the expectation is off (bolding mine):

> "For each target element, there is at least one device pixel inside the scrolling area of the viewport **whose color is different** when the element is focused from when it is not."

The addition of a border or increase in line weight for examples doesn't equal a change in color; it's adding something that wasn't there to begin with. As written this also seems like the entire background of the page changing when an element receives focus would pass which is not right.

There are also some general editorial changes that would help with plain language (e.g. use of the word "unicity") and punctuation consistency.

**Comment by Jym77** [comment]
Adding a border (or outline) is changing the colour of some pixels from white (background colour) to black (text colour) (or whatever other colours are used). So it is passing the rule.

The rule is indeed fairly lax in what it accepts as indicator. So is WCAG. A single pixel changing colour, or the full page changing colour, can be considered as a "visible indicator" by WCAG. We had quite some discussions on how big or small the indicator should be when writing the rule, and considered that we should stay on the safe side and accept almost anything in the rule (to avoid creating false positives). See discussions in the original PR for more details.

**Comment by ajanec01** [comment]
Hi @susanatx . The rule has changed a bit. The word color is no longer used in the expectation. I also agree with @Jym77 and think that the example you gave is covered by the rule. I'll work on the editorial changes but can you explain a bit more about the punctuation consistency?

**Comment by WilcoFiers** [comment]
As best I can tell this issue has been resolved.
